BUGFIX: Upgrade to Kubernetes 1.34 with CRI-O 1.34 (#131)
perses/helm-charts: docs/upgrade-guide.md

## Breaking Changes by Version

### Upgrading to 0.18.0

> This version introduces breaking changes to the `image` section. Some fields have been added.

Reviewer: ```suggestion
This version introduces breaking changes to the `image` section. A field has been added.
```
